packages/sources/anchorage/src/config/index.ts

```diff
@@ -0,0 +1,28 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+import { validator } from '@chainlink/external-adapter-framework/validation/utils'
+
+export const config = new AdapterConfig({
+  API_KEY: {
```

Comment:
do we need to support multiple API keys?

Reply:
Most likely no, at least not for this custodian. Still verifying that information

Reply:
Confirmed, one API_KEY for Anchorage